src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/runtime.Linux.Microsoft.NETCore.Runtime.CoreCLR.props

```diff
@@ -1,17 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_PlatformDoesNotSupportNiFiles Condition="'$(Platform)' == 'armel'">true</_PlatformDoesNotSupportNiFiles>
+    <_PlatformDoesNotSupportNiFiles Condition="'$(PackageTargetRuntime)' == 'ubuntu.14.04-arm'">true</_PlatformDoesNotSupportNiFiles>
```

Comment:
It appears to be supported for Ubuntu 16.10 arm.  Is that not true?  https://github.com/dotnet/coreclr/commit/89ba46aa6e113486428ac7528f662e3a0a5a55eb#diff-0656a30adba406c0c1d1b61c78a55025R20)